src/app/counter/counter-booking/counter-booking.component.html

```diff
@@ -1,8 +1,98 @@
+    </label>
+    <form class="form-inline">
+      <div class="form-group date-input">
+        <div class="input-group">
+          <input
+            name="minDate"
+            class="form-control"
+            placeholder="yyyy-mm-dd"
+            [(ngModel)]="minDate"
+            ngbDatepicker
+            [minDate]="getCurrentDate()"
+            #minD="ngbDatepicker"
+          />
+          <div class="input-group-append">
+            <button
+              class="btn btn-outline-secondary calendar"
+              (click)="minD.toggle()"
+              type="button"
+            >
+              <svg
+                width="1em"
+                height="1em"
+                viewBox="0 0 16 16"
+                class="bi bi-calendar3"
+                fill="currentColor"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  fill-rule="evenodd"
+                  d="M14 0H2a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V2a2 2 0 0 0-2-2zM1 3.857C1 3.384 1.448 3 2 3h12c.552 0 1 .384 1 .857v10.286c0 .473-.448.857-1 .857H2c-.552 0-1-.384-1-.857V3.857z"
+                />
+                <path
+                  fill-rule="evenodd"
+                  d="M6.5 7a1 1 0 1 0 0-2 1 1 0 0 0 0 2zm3 0a1 1 0 1 0 0-2 1 1 0 0 0 0 2zm3 0a1 1 0 1 0 0-2 1 1 0 0 0 0 2zm-9 3a1 1 0 1 0 0-2 1 1 0 0 0 0 2zm3 0a1 1 0 1 0 0-2 1 1 0 0 0 0 2zm3 0a1 1 0 1 0 0-2 1 1 0 0 0 0 2zm3 0a1 1 0 1 0 0-2 1 1 0 0 0 0 2zm-9 3a1 1 0 1 0 0-2 1 1 0 0 0 0 2zm3 0a1 1 0 1 0 0-2 1 1 0 0 0 0 2zm3 0a1 1 0 1 0 0-2 1 1 0 0 0 0 2z"
+                />
+              </svg>
+            </button>
+          </div>
+        </div>
+      </div>
+    </form>
+    <label class="col-form-label" for="inputDefault">
+      Latest Departure Date
+    </label>
+    <form class="form-inline">
+      <div class="form-group date-input">
+        <div class="input-group">
+          <input
+            name="maxDate"
+            class="form-control"
+            placeholder="yyyy-mm-dd"
+            [(ngModel)]="maxDate"
+            ngbDatepicker
+            [minDate]="getCurrentDate()"
+            #maxD="ngbDatepicker"
+          />
+          <div class="input-group-append">
+            <button
+              class="btn btn-outline-secondary calendar"
+              (click)="maxD.toggle()"
+              type="button"
+            >
+              <svg
+                width="1em"
+                height="1em"
+                viewBox="0 0 16 16"
+                class="bi bi-calendar3"
+                fill="currentColor"
+                xmlns="http://www.w3.org/2000/svg"
+              >
+                <path
+                  fill-rule="evenodd"
+                  d="M14 0H2a2 2 0 0 0-2 2v12a2 2 0 0 0 2 2h12a2 2 0 0 0 2-2V2a2 2 0 0 0-2-2zM1 3.857C1 3.384 1.448 3 2 3h12c.552 0 1 .384 1 .857v10.286c0 .473-.448.857-1 .857H2c-.552 0-1-.384-1-.857V3.857z"
+                />
+                <path
```

Comment:
One thing I noticed about these calendar images is they show up past the navbar. Should be an easy fix though.